miner/src/worker/cuckoo_simple.rs

```diff
@@ -162,15 +162,15 @@ impl Worker for CuckooSimple {
                     self.solve(&pow_hash, random());
```

Comment:
Not related, but it seems we don't need to clone the hash in the line above. Just use `self.pow_hash.as_ref()`.